### DISCUSSION TITLE
Download from channel4.com

### DISCUSSION THREAD

**Discussion by harryray33** [aclap-dev/video-downloadhelper#83] [Help and Q&A]
### Your question

I've been trying to download streaming videos from channel4.com.
I can't seem to do it with video-downloadhelper...
any ideas please?

### Addon & CoApp version and details

Firefox portable legacy 115, Windows 7, most recent addon and Coapp installed.

**Comment by paulrouget** [comment]
This uses DRM content. Nothing we can do about that sadly.
